### PR TITLE
fix: badges

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
 # examples
-Collection of example OpenRPC service definition files
```
Comment (shanejonas) [inline README.md]: should we leave this at the top to help npmjs.org descriptions?

Comment (BelfordZ) [reply README.md]: this should be done in the package.json though

**Comment by BelfordZ** [comment]
https://github.com/open-rpc/examples/pull/147

File: README.md (hunk 1 above)
Comment (BelfordZ) [reply README.md]: https://github.com/open-rpc/examples/pull/147

**Pull request merged**